jobs.py

```diff
@@ -120,6 +121,68 @@
         },
         "paths": ["tests/storage"],
         "markers": "quicktest",
+        "name_filter": "not linstor",
+    },
+    "linstor-main": {
+        "description": "tests the linstor storage driver, but avoids migrations and reboots",
+        "requirements": [
+            "A pool with at least 3 hosts.",
+            "An additional free disk on every host.",
+            "A small VM that can be imported on the SR.",
+        ],
+        "nb_pools": 1,
+        "params": {
+            "--vm": "single/small_vm",
+            "--sr-disk": "auto",
+        },
+        "paths": ["tests/storage/linstor"],
+        "markers": "(small_vm or no_vm) and not reboot and not quicktest",
+        "name_filter": "not migration",
+    },
+    "linstor-migrations": {
+        "description": "tests migrations with the linstor storage driver",
+        "requirements": [
+            "A pool with at least 3 hosts.",
+            "An additional free disk on every host.",
+            "A second pool with at least 1 host and a SR to receive VMs.",
+            "A small VM that can be imported on the SRs.",
+        ],
+        "nb_pools": 2,
+        "params": {
+            "--vm": "single/small_vm",
+            "--sr-disk": "auto",
+        },
+        "paths": ["tests/storage/linstor"],
+        "markers": "",
```

Comment:
is the empty field necessary?

Reply:
Why `markers` and not `name_filter`

Just asking its non blocking the merge IMHO

Reply:
Because I considered the use of `markers` to be common and "right", and the use of `name_filter` more of a hack to be used sparingly.